doc/changelogs/CHANGELOG_V18.md

```diff
@@ -41,6 +42,124 @@
   * [io.js](CHANGELOG_IOJS.md)
   * [Archive](CHANGELOG_ARCHIVE.md)
 
+<a id="18.9.0"></a>
+
+## 2022-09-05, Version 18.9.0 (Current), @RafaelGSS
```

Comment:
you'll need to update the dates, both here and in the commit message

Reply:
I'll use tomorrow's date then. Due to the CI and other stuff, I think I couldn't release it today. Does that make sense? Do I need to send any mailing info?

Reply:
> Do I need to send any mailing info?

not that I'm aware of, I don't think you need to worry about it.

> I'll use tomorrow's date then. 

👍 I also like to give myself the extra day bandwith to make sure CI is going to be green by then.